LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2021, Omnifinity, Kitlith & Erimel
+Copyright (c) 2023, Erimel
```

Comment:
I assumed previous copyright holders from the same license should have their name here as well. Is that not the case? Kinda looks like I’m claiming that all of it is mine (disregarding the notice in the readme ofc)